src/test/java/com/xceptance/neodymium/tests/NeodymiumTest.java

```diff
@@ -244,16 +296,20 @@ public void checkAnnotations(final Class<?> clazz, final Map<String, List<String
                 boolean expAnnotationFound = false;
                 for (String compAnnotation : compAnnotations)
                 {
-                    if (compAnnotation.equals(expAnnotation))
+                    if (compAnnotation.matches(".*" + expAnnotation + ".*"))
```

Comment:
is there cases when the annotation only matches the regex but is not equal to the expected string?

Reply:
Yes, I'm not 100% sure, why though. I guess it's JDK related but the versions with the strings is failing on my machine due to small differences. (e.g. "timeout=0" changed to "timeout=0L").